trade-finance-manager-api/api-tests/v1/deals/deals-submit.second-submission.api-test.js

```diff
@@ -654,7 +654,7 @@ describe('/v1/deals', () => {
 
         expect(status).toEqual(200);
         expect(body.submissionType).toEqual(CONSTANTS.DEALS.SUBMISSION_TYPE.MIN);
-        expect(typeof body.manualInclusionNoticeSubmissionDate).toEqual('number');
+        expect(typeof body.manualInclusionNoticeSubmissionDate).toEqual('string');
```

Comment:
Why is this changed to `string` please?

Reply:
now does .toString() so not sure why it was a number in the first place